br/br-use-overview.md

```diff
@@ -0,0 +1,101 @@
+
+## Recommended practices
+
+Before using TiDB backup and restore features, it is recommended that you understand the recommended backup and restore solutions.
+
+### How to back up data?
+
+**TiDB provides two types of backup. Which one should I use?** Full backup contains the full data of a cluster at a certain point in time. Log backup contains the data changes written to TiDB. It is recommended to use both types of backup at the same time:
+
+- **[Start log backup](/br/br-pitr-guide.md#start-log-backup)**: Run the `br log start` command to start the log backup task. After that, the task keeps running on all TiKV nodes and backs up TiDB data changes to the specified storage in small batches regularly.
+- **Perform [snapshot (full) backup](/br/br-snapshot-guide.md#back-up-snapshot-data) regularly**: Run the `br backup full` command to back up the snapshot of the cluster to the specified storage. For example, back up the cluster snapshot at 0:00 AM every day.
+
+### How to manage backup data?
+
+br command-line tool provides only basic backup and restore features, and does not support backup management. Therefore, you need to decide how to manage backup data on your own, which might involve the following questions?
+
+* Which backup storage system should I choose?
+* In which directory should I place the backup data during a backup task?
+* In what way should I organize the directory of the full backup data and log backup data?
+* How to handle the historical backup data in the storage system?
+
+The following sections will answer these questions one by one.
+
+**Choose a backup storage system**
+
+It is recommended that you store backup data to Amazon S3, Google Cloud Storage (GCS), or Azure Blob Storage. Using these systems, you do not need to worry about the backup capacity and bandwidth allocation.
+
+If the TiDB cluster is deployed in a self-built data center, the following practices are recommended:
+
+* Build [MinIO](https://docs.min.io/docs/minio-quickstart-guide.html) as the backup storage system, and use the S3 protocol to back up data to MinIO.
+* Mount Network File System (NFS, such as NAS) disks to br command-line tool and all TiKV instances, and use the POSIX file system interface to write backup data to the corresponding NFS directory.
+
+> **Note:**
+>
+> If no NFS is mounted to the br command-line tool or TiKV node, or if you use a storage system that supports Amazon S3, GCS, or Azure Blob Storage protocols, the data backed up by br command-line tool is generated at each TiKV node. **Note that this is not the recommended way to deploy the br command-line tool**, because the backup data are scattered in the local file system of each node. Collecting the backup data might result in data redundancy and operation and maintenance problems. Meanwhile, if you restore data directly before collecting the backup data, you will encounter the `SST file not found` error.
```

Comment:
```suggestion
> If you don't choose NFS or a storage system that supports Amazon S3, GCS, or Azure Blob Storage protocols, the data backed up is generated at each TiKV node. **Note that this is not the recommended way to use BR**, because collecting the backup data might result in data redundancy and operation and maintenance problems. 
```

Reply:
The change should be synchronized to Chinese. Otherwise, the Chinese and English are inconsistent.